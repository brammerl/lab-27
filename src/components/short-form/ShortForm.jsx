import React from 'react';
import PropTypes from 'prop-types';
import TextField from '@material-ui/core/TextField';
import Button from '@material-ui/core/Button';

const ShortForm = ({onSubmit, onChange, userName}) => (
  <form onSubmit={onSubmit}>
    <TextField label='Username' onChange={onChange} value={userName}/>
  </form>
)

export default ShortForm;