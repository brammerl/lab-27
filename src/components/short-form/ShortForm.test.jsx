import React from 'react';
import { shallow } from 'enzyme';
import ShortForm from './ShortForm';

describe('ShortForm Component', () => {
  it('renders the short form component', () => {
    const wrapper = shallow(<ShortForm/>)
    expect(wrapper).toMatchSnapshot();
  })
})