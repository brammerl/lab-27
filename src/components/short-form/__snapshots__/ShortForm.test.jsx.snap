// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShortForm Component renders the short form component 1`] = `
<form>
  <WithStyles(ForwardRef(TextField))
    label="Username"
  />
</form>
`;
